### DISCUSSION TITLE
Why are some download options grayed?

### DISCUSSION THREAD

**Discussion by GH6030** [aclap-dev/video-downloadhelper#1244] [Help and Q&A]
### Your question

Why are some download options grayed? As in this image, two of three MP4 options and two of three MKV options are grayed.
![Options](https://github.com/user-attachments/assets/d3d14424-24e6-4f00-a13f-5ca648061d9b)

### Addon & CoApp version and details

Version 9.1.0.44

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

You have sharp eyes.  I never noticed this before.

Those entries aren't gray.

![#01](https://github.com/user-attachments/assets/44e25264-f43a-4e97-b176-348a03c0024e)


1080p entries are blue & anything with a resolution higher than 1080p are green.  I think those are blue & green.  I'm not that great with colors.  In any case, it's just a way of accenting certain entries.  That's how it looks to me.  The rest of them are just a plain color.  They don't look gray to me but black.  Whatever the case, it's just a color & not really of any significance.  If the resolution is listed, VDH will download it.
